### DISCUSSION TITLE
Small number data not show when there is any large value in graph.

### DISCUSSION THREAD

**Discussion by JatinderSingh16** [plouc/nivo#1595] [Q&A] [answered]
Hi, I am facing one issue please guide me . when I show one higher number value bar 1000 and one bar is number 1 . its never showing count 1 bar . please help

**Comment by wyze** [comment, marked as answer]
You can use a symlog scale to achieve that. See our example: https://nivo.rocks/storybook/?path=/story/bar--with-symlog-scale
